### DISCUSSION TITLE
looking for mods for my project

### DISCUSSION THREAD

**Discussion by SakuraKitsunee** [Prodigy-Hacking/ProdigyMathGameHacking#693] [(A) Anything]
[Redacted]

**Comment by ghost** [comment]
bruh you literally copied what i wrote

**Reply by SakuraKitsunee** [reply]
i thought it was really well written so we are doing the same thing so i decided to use it, i can change it if you want?

**Reply by PatheticMustan** [reply]
Sad!

**Reply by ghost** [reply]
> i thought it was really well written so we are doing the same thing so i decided to use it, i can change it if you want?

eh its fine ig

**Comment by irhrhd** [comment]
Well give @NameIsA credit atleast

**Comment by ArcerionDev** [comment]
and as this is "not an official service of PMGH" and we "don't support giving out/logging into accounts for others for a variety of legitimate security reasons", you're gonna need to find another place to advertise this

**Comment by ghost** [comment]
I want to be a moderator!

**Reply by ghost** [reply]
their project violates the pmgh rules
